examples/index.html

```diff
@@ -652,17 +628,12 @@ <h2 id="examples_by_props_label">Examples By Properties and States</h2>
             <td><code>aria-multiselectable</code></td>
             <td><a href="listbox/listbox-rearrangeable.html">Listboxes with Rearrangeable Options</a></td>
           </tr>
-          <tr>
-            <td><code>aria-orientation</code></td>
-            <td><a href="slider/slider-2.html">Slider  with aria-orientation and aria-valuetext</a></td>
```

Comment:
Same aria-orientation bug mentioned above.

Reply:
@mcking65 
@carmacleod 
I will update the indexing to use the regex suggestion and see if that fixed the issue with `aria-orientation`.